checksum/Jenkinsfile-test

```diff
@@ -0,0 +1,82 @@
+library("tdr-jenkinslib")
+
+def versionTag = "v${env.BUILD_NUMBER}"
+def repo = "tdr-backend-checks"
+
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
+    }
+    stages {
+        stage("Build") {
+            agent {
+                ecs {
+                    inheritFrom "transfer-frontend"
+                }
+            }
+            steps {
+                script {
+                    tdr.reportStartOfBuildToGitHub(repo)
+                }
+                dir("checksum") {
+                    sh "sbt -no-colors assembly"
```

Comment:
Should this not be the following command to launch the tests: 'sh "sbt **test** -no-colors assembly'?

Reply:
sbt assembly runs the tests as well so it's a sort of a short cut.